### ISSUE TITLE
blob_timestamps=true prevents from restoring backup created with blob_timestamps=false

### ISSUE THREAD

**Issue by icemac** [collective/collective.recipe.backup#44]
I have a backup created with `collective.recipe.backup 3.x` thus with numbered blob-storage backups. 
Restoring the backup using `collective.recipe.backup 4.0` with `blob_timestamps=true` fails with:

```
ERROR: There are no backups in /<path>/var/blobstoragesnapshots.
ERROR: Halting execution: restoring blobstorages would fail.
```

There are actually two directories: `blobs.0` and `blobs.1` which the script does not recognize.
Is there a smooth way to upgrade to `blob_timestamps=true`?

**Comment by icemac** [comment]
Maybe an option for the `(snapshot)restore` script which changes the behaviour could help but this would require that restore is able to ignore directories of the other blob backup naming convention.

**Comment by icemac** [comment]
Maybe I confused some of the new options. Additionally I had activated `incremental_blobs = true`  and `archive_blob = true`. After deactivating them both the error vanished. So my question would rather be: How to upgrade smoothly to archived blobs?

**Comment by mauritsvanrees** [comment]
Sorry, I missed this question.

The restore script *should* be able to find backups created with all kinds of options. At least a combination of timestamps and older numbered backups should be fine. That scenario has tests. I expect different values for `incremental_blobs` to work fine too.

`archive_blob` might be the only one where this does not work. That *should* be fixable. I'll see what I can do.

Meanwhile, workaround would be basically what you already found: temporarily switch off `archive_blob`, or create a second buildout section with that option.

**Comment by mauritsvanrees** [comment]
Should be fixed with PR #47.

**Comment by mauritsvanrees** [comment]
I have release 4.0.1 with the fix.
